Sources/PackageGraph/ResolvedProduct.swift

```diff
@@ -69,8 +69,8 @@ public final class ResolvedProduct {
             return ResolvedTarget(
                 target: swiftTarget,
                 dependencies: targets.map { .target($0, conditions: []) },
-                defaultLocalization: .none, // safe since this is a derived product
-                platforms: .init(declared: [], derivedXCTestPlatformProvider: .none) // safe since this is a derived product
+                defaultLocalization: targets[0].defaultLocalization,
```

Comment:
No chance of the `targets` array being empty at this point?

Reply:
Eh, it's unclear to me how we'd get into a state where we're calling this function with a completely empty package, but we can afford to have a fallback path here, it's not a hot path or anything.